src/lwc/collapsibleSection/collapsibleSection.html

```diff
@@ -0,0 +1,16 @@
+<template>
```

Comment:
@bdvorachek - would it be worth consolidating this component and `utilExpandableSection` into one collapsible/expandable component? :\

Reply:
I think we can re-name collapsibleSection to utilExpandableSubSection, as the styling between these two is different. Maybe further down the line we combine the two and allow a "variant" parameter?

Reply:
This has been merged into utilExpandableSection, and a few updates were made to that component now.